examples/plot_johnson_lindenstrauss_bound.py

```diff
@@ -95,20 +95,23 @@
 from time import time
 import numpy as np
 import matplotlib
+import warnings
 import matplotlib.pyplot as plt
 from distutils.version import LooseVersion
 from sklearn.random_projection import johnson_lindenstrauss_min_dim
 from sklearn.random_projection import SparseRandomProjection
 from sklearn.datasets import fetch_20newsgroups_vectorized
 from sklearn.datasets import load_digits
 from sklearn.metrics.pairwise import euclidean_distances
+from sklearn.exceptions import DataDimensionalityWarning
 
 # `normed` is being deprecated in favor of `density` in histograms
 if LooseVersion(matplotlib.__version__) >= '2.1':
     density_param = {'density': True}
 else:
     density_param = {'normed': True}
 
+warnings.filterwarnings('ignore', category=DataDimensionalityWarning)
```

Comment:
I think this deserves a comment explaining why it's necessary / what the warning is about.